packages/nimble-components/src/select/index.ts

```diff
@@ -82,6 +82,12 @@ export class Select extends FormAssociatedSelect implements ErrorPattern {
     @attr({ attribute: 'filter-mode' })
     public filterMode: FilterMode = FilterMode.none;
 
+    /**
```

Comment:
@atmgrifter00 moved to comment as feedback was requested

@rajsite and @jattasNI would you mind taking a look at this draft PR to review the proposed API for the `Select` placeholder text? As I discussed with @jattasNI, this API is orthogonal to any work we define for the interaction to remove a selected value (which would ultimately display the placeholder text if configured that way).